index.js

```diff
@@ -1,13 +1,145 @@
-const express = require('express');
-const helmet = require('helmet');
+const express = require("express");
+const helmet = require("helmet");
+const knex = require("knex");
+const knexConfig = require("./knexfile");
+const db = knex(knexConfig.development);
 
 const server = express();
 
 server.use(express.json());
 server.use(helmet());
-
 // endpoints here
 
+server.post("/api/zoos", (req, res) => {
+  const newZoo = req.body;
+
+  db.insert(newZoo)
+    .into("zoos")
+    .then(ids => {
+      console.log(ids);
+      res.status(201).json(ids);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+server.post("/api/bears", (req, res) => {
+  const newBear = req.body;
+
+  db.insert(newBear)
+    .into("bears")
+    .then(ids => {
+      console.log(ids);
+      res.status(201).json(ids);
+    })
+    .catch(err => {
+      res.status(500).json(err);
+    });
+});
+// Get endpoints
+server.get("/api/zoos", async (req, res) => {
+  const allZoos = await db("zoos");
+  try {
+    res.status(200).json(allZoos);
+  } catch (error) {
+    res.status(500).json(error);
+  }
+});
+server.get("/api/bears", async (req, res) => {
+  const allbears = await db("bears");
+  try {
+    res.status(200).json(allZoos);
+  } catch (error) {
+    res.status(500).json(error);
+  }
+});
+
+// Curious if the catch() is still needed here, didn't add it because it seemed redundant
+server.get("/api/zoos/:id", (req, res) => {
+  db("zoos")
+    .where({ id: req.params.id })
+    .then(zoo => {
+      if (zoo) {
+        res.status(200).json(zoo);
+      } else {
+        res.status(404).json({ message: "Couldn't find that zoo" });
```

Comment:
When an ID is entered for a zoo (or bear in the other get by id endpoint), it is still returning an empty array with a 200 HTTP status code. You will probably want to async/await here and add a catch()